test/node/features/fixtures/express/scenarios/app.js

```diff
@@ -144,11 +144,11 @@ app.post('/features/unhandled', bodyParser.urlencoded(), function (req, res, nex
   // the request body is an object of feature flag name -> variant
   const featureFlags = Object.keys(req.body).map(name => ({ name, variant: req.body[name] }))
 
-  req.bugsnag.addFeatureFlags(featureFlags)
-  req.bugsnag.clearFeatureFlag('from config 3')
+  Bugsnag.addFeatureFlags(featureFlags)
+  Bugsnag.clearFeatureFlag('from config 3')
```

Comment:
This was failing due to the context loss from doing `bodyParser.urlencoded()` with the older version of the dependency. Bumping fixes it. Could also have used `middleware.runInContext` to restore it